tests/app/forms/test_fields.py

```diff
@@ -15,7 +15,15 @@ class TestFields(unittest.TestCase):
 
     def setUp(self):
         self.answer_store = AnswerStore()
-        self.metadata = parse_metadata({
+        self.schema_metadata = {
```

Comment:
You don't actually need `schema_metadata` for this test so it may confuse thing. It might be worth just removing the `parse_metadata` step from `self.metadata`.

Reply:
Sorted 👍 Ready for more :)